src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/ThrottlingRerouteSpec.groovy

```diff
@@ -50,55 +59,54 @@ class ThrottlingRerouteSpec extends HealthCheckSpecification {
 
         assumeTrue(swPairs.size() > 4, "Topology is too small to run this test")
         def flows = swPairs.take(5).collect { switchPair ->
-            def flow = flowHelperV2.randomFlow(switchPair)
-            flowHelperV2.addFlow(flow)
+            def flow = flowFactory.getRandom(switchPair)
             flow
         }
-        def flowPaths = flows.collect { northbound.getFlowPath(it.flowId) }
+        def flowPaths = flows.collect { it.retrieveAllEntityPaths() }
 
         when: "All flows break one by one"
-        def timeBeforeBreak = new Date().time
-        def brokenIsls = flowPaths.collect {
-            breakFlow(it, false)
-            //don't sleep here, since there is already an antiFlapMin delay between actual port downs
-        }
+        flowPaths.collect { breakFlow(it, false) }
+        // Don't sleep here, since there is already an antiFlapMin delay between actual port downs
         def rerouteTriggersEnd = new Date().time
         /*At this point all reroute triggers have happened. Save this time in order to calculate when the actual
         reroutes will happen (time triggers stopped + reroute delay seconds)*/
 
         then: "The oldest broken flow is still not rerouted before rerouteDelay run out"
         Wrappers.wait(rerouteDelay * 3) {
```

Comment:
can be replaced by 
`flow1.retrieveFlowHistory().getEntriesByType(REROUTE_ACTION)`
and the similar calls below

Reply:
Done